pandas/tests/plotting/test_misc.py

```diff
@@ -433,3 +433,99 @@ def test_dictionary_color(self):
         ax = df1.plot(kind="line", color=dic_color)
         colors = [rect.get_color() for rect in ax.get_lines()[0:2]]
         assert all(color == expected[index] for index, color in enumerate(colors))
+
+    @pytest.mark.slow
+    def test_has_externally_shared_axis(self):
```

Comment:
Rather than one large test can you split this into a few? Every comment block is probably a logical splitting point

Reply:
@WillAyd Ok, done.